Sources/SwiftProtobuf/Google_Protobuf_Struct.swift

```diff
@@ -304,16 +307,16 @@ public struct Google_Protobuf_Value: Message, Proto3Message, _MessageImplementat
     }
 
     public func _protoc_generated_traverse(visitor: Visitor) throws {
-        try kind.traverse(visitor: visitor, start:1, end: 7)
+        try kind?.traverse(visitor: visitor, start:1, end: 7)
     }
 
     // Storage ivars
-    private var kind = Google_Protobuf_Value.OneOf_Kind()
+    private var kind: Google_Protobuf_Value.OneOf_Kind?
```

Comment:
No, optionals are automatically initialized to nil. (We can probably shave off some codegen elsewhere by doing that.)